doc/installation/system/inifile.rst

```diff
@@ -83,6 +84,13 @@ slower but more robust and can be necessary in large redundant setups.
    row is otherwise not present". In this case setting ``PI_DB_SAFE_STORE``  to *True*
    might help.
 
+``PI_HASH_ALGO_LIST`` is a user-defined list of hash algorithms can be defined in the ``pi.cfg``.
+The context object will use the first algorithm listed in ``PI_HASH_ALGO_LIST`` when creating new hashes.
```

Comment:
The administrator might not know anything about CryptoContext here, we should just say that the first entry is used for hashing.

Reply:
fixed in 54e1e2f7b08297e5070bc3846df744a8a6f90e1f.